python/datashaper/datashaper/table_store/types.py

```diff
@@ -62,3 +62,11 @@ class TableContainer(Generic[T]):
     table: Table
     metadata: TableMetadata | None = None
     context: T | None = None
+
+
+@dataclass
+class ComplexVerbResult(Generic[T]):

```

Comment:
Does this actually need to be a separate concept from just a VerbResult? I.e., if we always allow multiple outputs, can we just skip the layer and consider that the standard option?

Reply:
I wanted to have minimal impact on existing verbs, which all emit 'TableContainer'. We could centralize this into 'VerbResult', but I'd rather do that as a follow-up PR

Reply:
Renamed this to 'VerbResult' since the Complex part wasn't necessary. Verbs can still return a TableContainer, but all of our built-in verbs emit a VerbResult now.